fiftyone/core/odm/utils.py

```diff
@@ -657,3 +657,28 @@ class DocumentRegistryError(Exception):
 
 
 _document_registry = DocumentRegistry()
+
+
+def load_dataset(_id=None, name=None):
+    """Loads the dataset from the database by _id or name."""
+    import fiftyone.core.odm as foo
+    import fiftyone.core.dataset as fod
+
+    if name:
+        return fod.Dataset(name, _create=False)
```

Comment:
would be nice to use `fod.load_dataset(name)` here and below in this function rather than `Dataset(name, _create=False)` to minimize the number of times that private kwargs are used